dev.restfox.Restfox.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: dev.restfox.Restfox
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+command: restfox
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+modules:
+  - name: restfox
+    build-commands:
+      - ar -o -x restfox.deb
+      - tar --extract --file data.tar.gz
+      - cp --preserve --recursive usr/* /app
+      - desktop-file-edit --set-key Exec --set-value "/app/bin/restfox %U" /app/share/applications/restfox.desktop
```

Comment:
```suggestion
      - desktop-file-edit --set-key Exec --set-value "restfox %U" /app/share/applications/restfox.desktop
```

Reply:
If you don't mind, I prefer to leave it with the path, because there could be at any point a few `restfox` in the path — since this tends to change as well with time.

The way it's right now, there won't be any mistakes — if something changes, it's this one thing.

Reply:
how could there be? It's inside of the sandbox. and its content doesn't vary without rebuild.

Reply:
The suggested change is to take a file named, `restfox`, resolved dynamically based on the current contents pf the `$PATH` variable — that's how I understand this.

What I eas explaining was that the `$PATH` variable can change over time as runtimes change/evolve; the content of the package may also be different, and also the manifest will change with time. As a result, there could be multiple versions of `restfox` that are accessible at any given moment.

However, the likelihood of this issue to happen is quite low...so I won't sweat it and adjust it.